packages/app/src/settings/actions/load-settings.action.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { DeviceType } from "@mudita/pure"
+import { createAsyncThunk } from "@reduxjs/toolkit"
+import isVersionGreater from "App/overview/helpers/is-version-greater"
+import { SettingsEvent } from "App/settings/constants"
+import { getSettings } from "App/settings/requests"
+import { loadBackupData } from "App/backup/actions"
+import {
+  checkAppForcedUpdateFlowToShow,
+  checkCollectingDataModalToShow,
+  checkAppUpdateFlowToShow,
+} from "App/modals-manager/actions"
+import { setSettings } from "App/settings/actions/set-settings.action"
+import logger from "App/__deprecated__/main/utils/logger"
+import getApplicationConfiguration from "App/__deprecated__/renderer/requests/get-application-configuration.request"
+import packageInfo from "../../../package.json"
+
+export const loadSettings = createAsyncThunk<void, void>(
+  SettingsEvent.LoadSettings,
+  async (_, { dispatch }) => {
+    let updateRequired = false
```

Comment:
Is there any plan for adding tests for `packages/app/src/settings/actions` folder?

Reply:
yes, it will be adding in [842](https://github.com/mudita/mudita-center/pull/842)